apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -78,6 +78,8 @@ export class NotificationService {
       case CaseNotificationType.APPEAL_JUDGES_ASSIGNED:
       case CaseNotificationType.APPEAL_CASE_FILES_UPDATED:
       case CaseNotificationType.CASE_FILES_UPDATED:
+      case CaseNotificationType.DISTRICT_COURT_JUDGE_ASSIGNED:
+      case CaseNotificationType.DISTRICT_COURT_REGISTRAR_ASSIGNED:
```

Comment:
Please remove the new notification types from this service. This service is only used by client facing controller endpoints. Since these notifications do not come from the client, but rather the case service, users should not be given permission to send them.